### PR TITLE
chore(deps): update some dependencies

### PR DESCRIPTION
**Pull request by rjzak**
A lot of crates have had breaking dependencies the past few months, trying to work through them.

### CODE REVIEW

**Comment by rjzak** [comment]
@haraldh I could use your wisdom...

What I don't understand is why this trait

https://github.com/enarx/enarx/blob/9577e8fcf26af58b99f63e40d0b530f90c834c50/src/backend/mod.rs#L120-L139

is no longer working for `sgx::signature::Signature` here:

https://github.com/enarx/enarx/blob/9577e8fcf26af58b99f63e40d0b530f90c834c50/src/backend/sgx/builder.rs#L135-L138

Even though these lines didn't change...

File: src/cli/sign.rs [hunk 1]
```diff
@@ -19,12 +19,12 @@ use anyhow::{anyhow, bail, Context, Result};
...
 use clap::Args;
 use p384::ecdsa::signature::Signer as _;
-use p384::ecdsa::SigningKey;
+use p384::ecdsa::{Signature, SigningKey};
 use p384::elliptic_curve::sec1::Coordinates;
 use p384::pkcs8::DecodePrivateKey;
 use p384::EncodedPoint;
 use sgx::crypto::{rcrypto::*, *};
-use sgx::signature::{Author, Body, Signature};
+use sgx::signature::{Author, Body};
```
Comment (haraldh) [inline src/cli/sign.rs:27]: Hmm.. you changed `Signature` and `unsafe impl ByteSized for Signature` does not apply anymore to `sgx::signature::Signature` then

**Review by haraldh — CHANGES_REQUESTED** [review]

**Comment by rjzak** [comment]
@haraldh I got https://github.com/enarx/enarx/pull/2561 working. But I'm not sure which is better:

* This PR:
```rust
const ES384: AlgorithmIdentifierOwned = AlgorithmIdentifierOwned {
    oid: ECDSA_WITH_SHA_384,
    parameters: None,
};
``` 

or other PR: https://github.com/enarx/enarx/pull/2561
```rust
const ES256: AlgorithmIdentifierRef<'static> = AlgorithmIdentifierRef {
    oid: ECDSA_WITH_SHA_256,
    parameters: None,
};
```

I'm guessing the second one, which I un-drafted.

**Comment by rjzak** [comment]
Superseded by #2561

**Pull request closed without merging**